[Bug][UT] CreateTableTest is easily failed

**Describe the bug**
run fe ut, and we could easily see the following failure on CreateTableTest:

[ERROR] Tests run: 3, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 33.192 s <<< FAILURE! - in org.apache.doris.catalog.CreateTableTest
[ERROR] testAbnormal(org.apache.doris.catalog.CreateTableTest)  Time elapsed: 10.068 s  <<< FAILURE!
junit.framework.AssertionFailedError: errCode = 2, detailMessage = Failed to create partition[atbl6]. Timeout. Unfinished mark: 10002=10014
	at org.apache.doris.catalog.CreateTableTest.testAbnormal(CreateTableTest.java:247)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   CreateTableTest.testAbnormal:247 errCode = 2, detailMessage = Failed to create partition[atbl6]. Timeout. Unfinished mark: 10002=10014
[INFO] 
[ERROR] Tests run: 3, Failures: 1, Errors: 0, Skipped: 0

This is because we don't init other be port at first, and the default value for port is -1 
**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - Browser [e.g. stock browser, safari]
 - Version [e.g. 22]

**Additional context**
Add any other context about the problem here.

--- a/fe/fe-core/src/test/java/org/apache/doris/utframe/UtFrameUtils.java
+++ b/fe/fe-core/src/test/java/org/apache/doris/utframe/UtFrameUtils.java
@@ -190,6 +190,9 @@ public class UtFrameUtils {
         be.setDisks(ImmutableMap.copyOf(disks));
         be.setAlive(true);
         be.setOwnerClusterName(SystemInfoService.DEFAULT_CLUSTER);
+        be.setBePort(be_thrift_port);
+        be.setHttpPort(be_http_port);
+        be.setBeRpcPort(be_brpc_port);
         Catalog.getCurrentSystemInfo().addBackend(be);
     }
 

